packages/utils/src/Signers.ts

```diff
@@ -0,0 +1,454 @@
+/**
+ * Copyright (c) 2018-2023, BOTLabs GmbH.
+ *
+ * This source code is licensed under the BSD 4-Clause "Original" license
+ * found in the LICENSE file in the root directory of this source tree.
+ */
+
+import type { Signer } from '@polkadot/api/types/index.js'
+import { decodePair } from '@polkadot/keyring/pair/decode'
+import {
+  hexToU8a,
+  u8aConcat,
+  u8aIsWrapped,
+  u8aToHex,
+  u8aWrapBytes,
+} from '@polkadot/util'
+import {
+  blake2AsU8a,
+  encodeAddress,
+  secp256k1Sign,
+} from '@polkadot/util-crypto'
+import type { Keypair } from '@polkadot/util-crypto/types'
+
+import {
+  createSigner as es256kSigner,
+  cryptosuite as es256kSuite,
+} from '@kiltprotocol/es256k-jcs-2023'
+import {
+  createSigner as ed25519Signer,
+  cryptosuite as ed25519Suite,
+} from '@kiltprotocol/eddsa-jcs-2022'
+import {
+  cryptosuite as sr25519Suite,
+  createSigner as sr25519Signer,
```

Comment:
Can we declare this where we use it

Reply:
I wanted to make these available, but obviously they need to be identifiable by the algorithm they use. What are you suggesting?

Reply:
I suggest if you have a better idea (like exporting both under cryptosuite and createSigner on an object, e.g., `ed25519Suite`) we make a ticket and implement it on a feature branch forked from develop